Source/WebCore/platform/graphics/avfoundation/objc/SourceBufferParserAVFObjC.mm

```diff
@@ -65,10 +65,10 @@ @interface AVContentKeySpecifier (WebCorePrivate)
 #endif
 
 @interface WebAVStreamDataParserListener : NSObject<AVStreamDataParserOutputHandling> {
-    WebCore::SourceBufferParserAVFObjC* _parent;
+    WeakPtr<WebCore::SourceBufferParserAVFObjC> _parent;
     AVStreamDataParser* _parser;
```

Comment:
How about this one?

Reply:
Hmm static analysis didn't complain about this one..

Reply:
I guess because this is an ObjC type. I can use an ObjC weak ptr for safety but our analyzer is not yet able to catch this.

Reply:
oh yeah, we need a checker for `RetainPtr`.